---
title: "Data"
parent: "Test"
has_toc: true
nav_order: 1
---



## ADeditome provides the genomic landscape of A-to-I RNA editing in Alzheimer's disease.

A-to-I RNA editing, contributing to nearly 90% of all editing events in human, has been reported to involve in the pathogenesis of Alzheimer's disease (AD) due to its roles in brain development and immune regulation, such as the deficient editing of GluA2 Q/R related to cell death and memory loss. Currently, there are urgent needs for the systematic annotations of A-to-I RNA editing events in AD. Here, we built ADeditome, the annotation database of A-to-I RNA editing in AD available at https://ccsm.uth.edu/ADeditome, aiming to provide a resource and reference for functional annotation of A-to-I RNA editing in AD to identify therapeutically targetable genes in an individual. We detected 1676 363 editing sites in 1524 samples across nine brain regions from ROSMAP, MayoRNAseq and MSBB. For these editing events, we performed multiple functional annotations including identification of specific and disease stage associated editing events and the influence of editing events on gene expression, protein recoding, alternative splicing and miRNA regulation for all the genes, especially for AD-related genes in order to explore the pathology of AD. Combing all the analysis results, we found 108 010 and 26 168 editing events which may promote or inhibit AD progression, respectively. We also found 5582 brain region-specific editing events with potentially dual roles in AD across different brain regions. ADeditome will be a unique resource for AD and drug research communities to identify therapeutically targetable editing events. Significance: ADeditome is the first comprehensive resource of the functional genomics of individual A-to-I RNA editing events in AD, which will be useful for many researchers in the fields of AD pathology, precision medicine, and therapeutic researches.

URL: https://ccsm.uth.edu/ADeditome,


## Automatic morphometry in Alzheimer's disease and mild cognitive impairment.

This paper presents a novel, publicly available repository of anatomically segmented brain images of healthy subjects as well as patients with mild cognitive impairment and Alzheimer's disease. The underlying magnetic resonance images have been obtained from the Alzheimer's Disease Neuroimaging Initiative (ADNI) database. T1-weighted screening and baseline images (1.5T and 3T) have been processed with the multi-atlas based MAPER procedure, resulting in labels for 83 regions covering the whole brain in 816 subjects. Selected segmentations were subjected to visual assessment. The segmentations are self-consistent, as evidenced by strong agreement between segmentations of paired images acquired at different field strengths (Jaccard coefficient: 0.802+-0.0146). Morphometric comparisons between diagnostic groups (normal; stable mild cognitive impairment; mild cognitive impairment with progression to Alzheimer's disease; Alzheimer's disease) showed highly significant group differences for individual regions, the majority of which were located in the temporal lobe. Additionally, significant effects were seen in the parietal lobe. Increased left/right asymmetry was found in posterior cortical regions. An automatically derived white-matter hypointensities index was found to be a suitable means of quantifying white-matter disease. This repository of segmentations is a potentially valuable resource to researchers working with ADNI data.

URL:


## A harmonized longitudinal biomarkers and cognition database for assessing the natural history of preclinical Alzheimer's disease from young adulthood and for designing prevention trials.

INTRODUCTION: Large longitudinal biomarkers database focusing on middle age is needed for Alzheimer's disease (AD) prevention. METHODS: Data for cerebrospinal fluid analytes, molecular imaging of cerebral fibrillar beta-amyloid with positron emission tomography, magnetic resonance imaging-based brain structures, and clinical/cognitive outcomes were harmonized across eight AD biomarker studies. Statistical power was estimated. RESULTS: The harmonized database included 7779 participants with clinical/cognitive data: 3542 were 18~65 years at the baseline, 5865 had longitudinal cognitive data for a median of 4.7 years, 2473 participated in the cerebrospinal fluid studies (906 had longitudinal data), 2496 participated in the magnetic resonance imaging studies (1283 had longitudinal data), and 1498 participated in the positron emission tomography amyloid studies (849 had longitudinal data). The database provides adequate power for detecting early biomarker changes, and demonstrates the feasibility of AD prevention trials on middle-aged individuals. DISCUSSION: The harmonized database is an optimum resource to design AD prevention trials decades before symptomatic onset.

URL:


## A single-cell and spatial RNA-seq database for Alzheimer's disease (ssREAD).

Alzheimer's Disease (AD) pathology has been increasingly explored through single-cell and single-nucleus RNA-sequencing (scRNA-seq & snRNA-seq) and spatial transcriptomics (ST). However, the surge in data demands a comprehensive, user-friendly repository. Addressing this, we introduce a single-cell and spatial RNA-seq database for Alzheimer's disease (ssREAD). It offers a broader spectrum of AD-related datasets, an optimized analytical pipeline, and improved usability. The database encompasses 1,053 samples (277 integrated datasets) from 67 AD-related scRNA-seq & snRNA-seq studies, totaling 7,332,202 cells. Additionally, it archives 381 ST datasets from 18 human and mouse brain studies. Each dataset is annotated with details such as species, gender, brain region, disease/control status, age, and AD Braak stages. ssREAD also provides an analysis suite for cell clustering, identification of differentially expressed and spatially variable genes, cell-type-specific marker genes and regulons, and spot deconvolution for integrative analysis. ssREAD is freely available at https://bmblx.bmi.osumc.edu/ssread/ .

URL: https://bmblx.bmi.osumc.edu/ssread/


## AlzRiskMR database: an online database for the impact of exposure factors on Alzheimer's disease.

In view of great difficulties in the pathogenesis analysis of Alzheimer's disease (AD) presently, profiling the modifiable risk factors is crucial for early detection and intervention of AD. However, the causal associations among them have yet to be identified, and the effective integration and application of these data also remain considerable challenges due to the lack of efficient collection and analysis procedures. To address this issue, we performed comprehensive analyses by two-sample Mendelian randomization (2SMR) and established the AlzRiskMR database (https://github.com/SDBMC/RiskFactors2AD). Four 2SMR analysis methods, including inverse variance weighted (IVW), MR-Egger, weighted median, and weighted mode, were used for the complementary calculation to test the reliability of the results. The database currently comprises 1870 sets of data of Genome-Wide Association Studies (GWAS) from the MR-Base and NHGRI-EBI GWAS Catalog database. AlzRiskMR database not only estimates causal associations between modifiable risk factors and AD but also offers a useful and timely resource for early intervention of AD development incidence.

URL: https://github.com/SDBMC/RiskFactors2AD


## The European DTI Study on Dementia - A multicenter DTI and MRI study on Alzheimer's disease and Mild Cognitive Impairment.

The European DTI Study on Dementia (EDSD) is a multicenter framework created to study the diagnostic accuracy and inter-site variability of DTI-derived markers in patients with manifest and prodromal Alzheimer's disease (AD). The dynamically growing database presently includes 493 DTI, 512 T1-weighted MRI, and 300 FLAIR scans from patients with AD dementia, patients with Mild Cognitive Impairment (MCI) and matched Healthy Controls, acquired on 13 different scanner platforms. The imaging data is publicly available, along with the subjects' demographic and clinical characterization. Detailed neuropsychological information, cerebrospinal fluid information on biomarkers and clinical follow-up diagnoses are included for a subset of subjects. This paper describes the rationale and structure of the EDSD, summarizes the available data, and explains how to gain access to the database. The EDSD is a useful database for researchers seeking to investigate the contribution of DTI to dementia diagnostics.

URL:


## A multi-site, multi-participant magnetoencephalography resting-state dataset to study dementia: The BioFIND dataset.

Early detection of Alzheimer's Disease (AD) is vital to reduce the burden of dementia and for developing effective treatments. Neuroimaging can detect early brain changes, such as hippocampal atrophy in Mild Cognitive Impairment (MCI), a prodromal state of AD. However, selecting the most informative imaging features by machine-learning requires many cases. While large publically-available datasets of people with dementia or prodromal disease exist for Magnetic Resonance Imaging (MRI), comparable datasets are missing for Magnetoencephalography (MEG). MEG offers advantages in its millisecond resolution, revealing physiological changes in brain oscillations or connectivity before structural changes are evident with MRI. We introduce a MEG dataset with 324 individuals: patients with MCI and healthy controls. Their brain activity was recorded while resting with eyes closed, using a 306-channel MEG scanner at one of two sites (Madrid or Cambridge), enabling tests of generalization across sites. A T1-weighted MRI is provided to assist source localisation. The MEG and MRI data are formatted according to international BIDS standards and analysed freely on the DPUK platform (https://portal.dementiasplatform.uk/Apply). Here, we describe this dataset in detail, report some example (benchmark) analyses, and consider its limitations and future directions.

URL: https://portal.dementiasplatform.uk/Apply


## ExonSkipAD provides the functional genomic landscape of exon skipping events in Alzheimer's disease.

Exon skipping (ES), the most common alternative splicing event, has been reported to contribute to diverse human diseases due to the loss of functional domains/sites or frameshifting of the open reading frame (ORF) and noticed as therapeutic targets. Accumulating transcriptomic studies of aging brains show the splicing disruption is a widespread hallmark of neurodegenerative diseases such as Alzheimer's disease (AD). Here, we built ExonSkipAD, the ES annotation database aiming to provide a resource/reference for functional annotation of ES events in AD and identify therapeutic targets in exon units. We identified 16 414 genes that have ~156 K, ~ 69 K, ~ 231 K ES events from the three representative AD cohorts of ROSMAP, MSBB and Mayo, respectively. For these ES events, we performed multiple functional annotations relating to ES mechanisms or downstream. Specifically, through the functional feature retention studies followed by the open reading frames (ORFs), we identified 275 important cellular regulators that might lose their cellular regulator roles due to exon skipping in AD. ExonSkipAD provides twelve categories of annotations: gene summary, gene structures and expression levels, exon skipping events with PSIs, ORF annotation, exon skipping events in the canonical protein sequence, 3'-UTR located exon skipping events lost miRNA-binding sites, SNversus in the skipped exons with a depth of coverage, AD stage-associated exon skipping events, splicing quantitative trait loci (sQTLs) in the skipped exons, correlation with RNA-binding proteins, and related drugs & diseases. ExonSkipAD will be a unique resource of transcriptomic diversity research for understanding the mechanisms of neurodegenerative disease development and identifying potential therapeutic targets in AD. Significance AS the first comprehensive resource of the functional genomics of the alternative splicing events in AD, ExonSkipAD will be useful for many researchers in the fields of pathology, AD genomics and precision medicine, and pharmaceutical and therapeutic researches.

URL:


## NIAGADS Alzheimer's GenomicsDB: A resource for exploring Alzheimer's disease genetic and genomic knowledge.

INTRODUCTION: The National Institute on Aging Genetics of Alzheimer's Disease Data Storage Site Alzheimer's Genomics Database (GenomicsDB) is a public knowledge base of Alzheimer's disease (AD) genetic datasets and genomic annotations. METHODS: GenomicsDB uses a custom systems architecture to adopt and enforce rigorous standards that facilitate harmonization of AD-relevant genome-wide association study summary statistics datasets with functional annotations, including over 230 million annotated variants from the AD Sequencing Project. RESULTS: GenomicsDB generates interactive reports compiled from the harmonized datasets and annotations. These reports contextualize AD-risk associations in a broader functional genomic setting and summarize them in the context of functionally annotated genes and variants. DISCUSSION: Created to make AD-genetics knowledge more accessible to AD researchers, the GenomicsDB is designed to guide users unfamiliar with genetic data in not only exploring but also interpreting this ever-growing volume of data. Scalable and interoperable with other genomics resources using data technology standards, the GenomicsDB can serve as a central hub for research and data analysis on AD and related dementias. HIGHLIGHTS: The National Institute on Aging Genetics of Alzheimer's Disease Data Storage Site (NIAGADS) offers to the public a unique, disease-centric collection of AD-relevant GWAS summary statistics datasets. Interpreting these data is challenging and requires significant bioinformatics expertise to standardize datasets and harmonize them with functional annotations on genome-wide scales. The NIAGADS Alzheimer's GenomicsDB helps overcome these challenges by providing a user-friendly public knowledge base for AD-relevant genetics that shares harmonized, annotated summary statistics datasets from the NIAGADS repository in an interpretable, easily searchable format.

URL:


## Development of a unified clinical trial database for Alzheimer's disease.

INTRODUCTION: Data obtained in completed Alzheimer's disease (AD) clinical trials can inform decision making for future trials. Recognizing the importance of sharing these data, the Coalition Against Major Diseases created an Online Data Repository for AD (CODR-AD) with the aim of supporting accelerated drug development. The aim of this study was to build an open access, standardized database from control arm data collected across many clinical trials. METHODS: Comprehensive AD-specific data standards were developed to enable the pooling of data from different sources. Nine member organizations contributed patient-level data from 24 clinical trials of AD treatments. RESULTS: CODR-AD consists of control arm pooled and standardized data from 24 trials currently numbered at 6500 subjects; Alzheimer's Disease Assessment Scale-cognitive subscale 11 is the main outcome and specific covariates are also included. DISCUSSION: CODR-AD represents a unique integrated standardized clinical trials database available to qualified researchers. The pooling of data across studies facilitates a more comprehensive understanding of disease heterogeneity.

URL:


## Virginia Memory Project: Using the Healthy Brain Initiative Roadmap to design a statewide dementia registry.

INTRODUCTION: The Virginia Memory Project (VMP) is a statewide epidemiological registry for Alzheimer's disease and related disorders (ADRD) and other neurodegenerative conditions. It aims to support dementia research, policy, and care by leveraging the Centers for Disease Control (CDC) Healthy Brain Initiative (HBI) Roadmap. METHODS: To capture comprehensive data, the VMP integrates self-enrollment and automatic enrollment using Virginia's All-Payer Claims Database (APCD). It also adapts Behavioral Risk Factors Surveillance Survey (BRFSS) modules for self-reported cognitive and caregiving data, offering connections to research, clinical services, and education. RESULTS: Virginia successfully codified the VMP in the 2024 general assembly session. DISCUSSION: The VMP demonstrates a novel approach to Alzheimer's Disease and Related Disorders (ADRD) surveillance by combining traditional registry functions with community engagement and workforce development. Future efforts will focus on increasing enrollment, especially among underrepresented groups, to enhance data-driven dementia policy and care in Virginia. HIGHLIGHTS: Integrated the Healthy Brain Initiative (HBI) domains into the newest statewide epidemiological dementia registry in the Commonwealth of Virginia. Collected data and identified gaps in the current research related to dementia and Alzheimer's related diseases. Aimed to mitigate barriers to dementia registry enrollment by identifying significant underdiagnosis and underrepresentation of racial and ethnic minority groups. Developed solutions to alleviate the current data and enrollment disparities and to connect individuals to research, physicians, and community groups.

URL:


## SCAN: Spatiotemporal Cloud Atlas for Neural cells.

The nervous system is one of the most complicated and enigmatic systems within the animal kingdom. Recently, the emergence and development of spatial transcriptomics (ST) and single-cell RNA sequencing (scRNA-seq) technologies have provided an unprecedented ability to systematically decipher the cellular heterogeneity and spatial locations of the nervous system from multiple unbiased aspects. However, efficiently integrating, presenting and analyzing massive multiomic data remains a huge challenge. Here, we manually collected and comprehensively analyzed high-quality scRNA-seq and ST data from the nervous system, covering 10 679 684 cells. In addition, multi-omic datasets from more than 900 species were included for extensive data mining from an evolutionary perspective. Furthermore, over 100 neurological diseases (e.g. Alzheimer's disease, Parkinson's disease, Down syndrome) were systematically analyzed for high-throughput screening of putative biomarkers. Differential expression patterns across developmental time points, cell types and ST spots were discerned and subsequently subjected to extensive interpretation. To provide researchers with efficient data exploration, we created a new database with interactive interfaces and integrated functions called the Spatiotemporal Cloud Atlas for Neural cells (SCAN), freely accessible at http://47.98.139.124:8799 or http://scanatlas.net. SCAN will benefit the neuroscience research community to better exploit the spatiotemporal atlas of the neural system and promote the development of diagnostic strategies for various neurological disorders.

URL: http://47.98.139.124:8799


## A globally diverse reference alignment and panel for imputation of mitochondrial DNA variants.

BACKGROUND: Variation in mitochondrial DNA (mtDNA) identified by genotyping microarrays or by sequencing only the hypervariable regions of the genome may be insufficient to reliably assign mitochondrial genomes to phylogenetic lineages or haplogroups. This lack of resolution can limit functional and clinical interpretation of a substantial body of existing mtDNA data. To address this limitation, we developed and evaluated a large, curated reference alignment of complete mtDNA sequences as part of a pipeline for imputing missing mtDNA single nucleotide variants (mtSNVs). We call our reference alignment and pipeline MitoImpute. RESULTS: We aligned the sequences of 36,960 complete human mitochondrial genomes downloaded from GenBank, filtered and controlled for quality. These sequences were reformatted for use in imputation software, IMPUTE2. We assessed the imputation accuracy of MitoImpute by measuring haplogroup and genotype concordance in data from the 1000 Genomes Project and the Alzheimer's Disease Neuroimaging Initiative (ADNI). The mean improvement of haplogroup assignment in the 1000 Genomes samples was 42.7% (Matthew's correlation coefficient = 0.64). In the ADNI cohort, we imputed missing single nucleotide variants. CONCLUSION: These results show that our reference alignment and panel can be used to impute missing mtSNVs in existing data obtained from using microarrays, thereby broadening the scope of functional and clinical investigation of mtDNA. This improvement may be particularly useful in studies where participants have been recruited over time and mtDNA data obtained using different methods, enabling better integration of early data collected using less accurate methods with more recent sequence data.

URL:


## The Bio-Hermes Study: Biomarker database developed to investigate blood-based and digital biomarkers in community-based, diverse populations clinically screened for Alzheimer's disease.

INTRODUCTION: Alzheimer's disease (AD) trial participants are often screened for eligibility by brain amyloid positron emission tomography/cerebrospinal fluid (PET/CSF), which is inefficient as many are not amyloid positive. Use of blood-based biomarkers may reduce screen failures. METHODS: We recruited 755 non-Hispanic White, 115 Hispanic, 112 non-Hispanic Black, and 19 other minority participants across groups of cognitively normal (n = 417), mild cognitive impairment (n = 312), or mild AD (n = 272) participants. Plasma amyloid beta (Abeta)40, Abeta42, Abeta42/Abeta40, total tau, phosphorylated tau (p-tau)181, and p-tau217 were measured; amyloid PET/CSF (n = 956) determined amyloid positivity. Clinical, blood biomarker, and ethnicity/race differences associated with amyloid status were evaluated. RESULTS: Greater impairment, older age, and carrying an apolipoprotein E (apoE) epsilon4 allele were associated with greater amyloid burden. Areas under the receiver operating characteristic curve for amyloid status of plasma Abeta42/Abeta40, p-tau181, and p-tau217 with amyloid positivity were >= 0.7117 for all ethnoracial groups (p-tau217, >=0.8128). Age and apoE epsilon4 adjustments and imputation of biomarker values outside limit of quantitation provided small improvement in predictive power. DISCUSSION: Blood-based biomarkers are highly associated with amyloid PET/CSF results in diverse populations enrolled at clinical trial sites. HIGHLIGHTS: Amyloid beta (Abeta)42/Abeta40, phosphorylated tau (p-tau)181, and p-tau 217 blood-based biomarkers predicted brain amyloid positivity. P-tau 217 was the strongest predictor of brain amyloid positivity. Biomarkers from diverse ethnic, racial, and clinical cohorts predicted brain amyloid positivity. Community-based populations have similar Alzheimer's disease (AD) biomarker levels as other populations. A prescreen process with blood-based assays may reduce the number of AD trial screen failures.

URL:


## Building an integrated neurodegenerative disease database at an academic health center.

BACKGROUND: It is becoming increasingly important to study common and distinct etiologies, clinical and pathological features, and mechanisms related to neurodegenerative diseases such as Alzheimer's disease, Parkinson's disease, amyotrophic lateral sclerosis, and frontotemporal lobar degeneration. These comparative studies rely on powerful database tools to quickly generate data sets that match diverse and complementary criteria set by them. METHODS: In this article, we present a novel integrated neurodegenerative disease (INDD) database, which was developed at the University of Pennsylvania (Penn) with the help of a consortium of Penn investigators. Because the work of these investigators are based on Alzheimer's disease, Parkinson's disease, amyotrophic lateral sclerosis, and frontotemporal lobar degeneration, it allowed us to achieve the goal of developing an INDD database for these major neurodegenerative disorders. We used the Microsoft SQL server as a platform, with built-in "backwards" functionality to provide Access as a frontend client to interface with the database. We used PHP Hypertext Preprocessor to create the "frontend" web interface and then used a master lookup table to integrate individual neurodegenerative disease databases. We also present methods of data entry, database security, database backups, and database audit trails for this INDD database. RESULTS: Using the INDD database, we compared the results of a biomarker study with those using an alternative approach by querying individual databases separately. CONCLUSIONS: We have demonstrated that the Penn INDD database has the ability to query multiple database tables from a single console with high accuracy and reliability. The INDD database provides a powerful tool for generating data sets in comparative studies on several neurodegenerative diseases.

URL:


## ZEBRA: a hierarchically integrated gene expression atlas of the murine and human brain at single-cell resolution.

The molecular causes and mechanisms of neurodegenerative diseases remain poorly understood. A growing number of single-cell studies have implicated various neural, glial, and immune cell subtypes to affect the mammalian central nervous system in many age-related disorders. Integrating this body of transcriptomic evidence into a comprehensive and reproducible framework poses several computational challenges. Here, we introduce ZEBRA, a large single-cell and single-nucleus RNA-seq database. ZEBRA integrates and normalizes gene expression and metadata from 33 studies, encompassing 4.2 million human and mouse brain cells sampled from 39 brain regions. It incorporates samples from patients with neurodegenerative diseases like Alzheimer's disease, Parkinson's disease, and Multiple sclerosis, as well as samples from relevant mouse models. We employed scVI, a deep probabilistic auto-encoder model, to integrate the samples and curated both cell and sample metadata for downstream analysis. ZEBRA allows for cell-type and disease-specific markers to be explored and compared between sample conditions and brain regions, a cell composition analysis, and gene-wise feature mappings. Our comprehensive molecular database facilitates the generation of data-driven hypotheses, enhancing our understanding of mammalian brain function during aging and disease. The data sets, along with an interactive database are freely available at https://www.ccb.uni-saarland.de/zebra.

URL: https://www.ccb.uni-saarland.de/zebra.


## The Washington University Central Neuroimaging Data Archive.

Since the early 2000's, much of the neuroimaging work at Washington University (WU) has been facilitated by the Central Neuroimaging Data Archive (CNDA), an XNAT-based imaging informatics system. The CNDA is uniquely related to XNAT, as it served as the original codebase for the XNAT open source platform. The CNDA hosts data acquired in over 1000 research studies, encompassing 36,000 subjects and more than 60,000 imaging sessions. Most imaging modalities used in modern human research are represented in the CNDA, including magnetic resonance (MR), positron emission tomography (PET), computed tomography (CT), nuclear medicine (NM), computed radiography (CR), digital radiography (DX), and ultrasound (US). However, the majority of the imaging data in the CNDA are MR and PET of the human brain. Currently, about 20% of the total imaging data in the CNDA is available by request to external researchers. CNDA's available data includes large sets of imaging sessions and in some cases clinical, psychometric, tissue, or genetic data acquired in the study of Alzheimer's disease, brain metabolism, cancer, HIV, sickle cell anemia, and Tourette syndrome.

URL:


## VCPA: genomic variant calling pipeline and data management tool for Alzheimer's Disease Sequencing Project.

SUMMARY: We report VCPA, our SNP/Indel Variant Calling Pipeline and data management tool used for the analysis of whole genome and exome sequencing (WGS/WES) for the Alzheimer's Disease Sequencing Project. VCPA consists of two independent but linkable components: pipeline and tracking database. The pipeline, implemented using the Workflow Description Language and fully optimized for the Amazon elastic compute cloud environment, includes steps from aligning raw sequence reads to variant calling using GATK. The tracking database allows users to view job running status in real time and visualize >100 quality metrics per genome. VCPA is functionally equivalent to the CCDG/TOPMed pipeline. Users can use the pipeline and the dockerized database to process large WGS/WES datasets on Amazon cloud with minimal configuration. AVAILABILITY AND IMPLEMENTATION: VCPA is released under the MIT license and is available for academic and nonprofit use for free. The pipeline source code and step-by-step instructions are available from the National Institute on Aging Genetics of Alzheimer's Disease Data Storage Site (http://www.niagads.org/VCPA). SUPPLEMENTARY INFORMATION: Supplementary data are available at Bioinformatics online.

URL: http://www.niagads.org/VCPA


## Compilation of reported protein changes in the brain in Alzheimer's disease.

Proteomic studies of human Alzheimer's disease brain tissue have potential to identify protein changes that drive disease, and to identify new drug targets. Here, we analyse 38 published Alzheimer's disease proteomic studies, generating a map of protein changes in human brain tissue across thirteen brain regions, three disease stages (preclinical Alzheimer's disease, mild cognitive impairment, advanced Alzheimer's disease), and proteins enriched in amyloid plaques, neurofibrillary tangles, and cerebral amyloid angiopathy. Our dataset is compiled into a searchable database (NeuroPro). We found 848 proteins were consistently altered in 5 or more studies. Comparison of protein changes in early-stage and advanced Alzheimer's disease revealed proteins associated with synapse, vesicle, and lysosomal pathways show change early in disease, but widespread changes in mitochondrial associated protein expression change are only seen in advanced Alzheimer's disease. Protein changes were similar for brain regions considered vulnerable and regions considered resistant. This resource provides insight into Alzheimer's disease brain protein changes and highlights proteins of interest for further study.

URL:


## A platform for discovery: The University of Pennsylvania Integrated Neurodegenerative Disease Biobank.

Neurodegenerative diseases (NDs) are defined by the accumulation of abnormal protein deposits in the central nervous system (CNS), and only neuropathological examination enables a definitive diagnosis. Brain banks and their associated scientific programs have shaped the actual knowledge of NDs, identifying and characterizing the CNS deposits that define new diseases, formulating staging schemes, and establishing correlations between neuropathological changes and clinical features. However, brain banks have evolved to accommodate the banking of biofluids as well as DNA and RNA samples. Moreover, the value of biobanks is greatly enhanced if they link all the multidimensional clinical and laboratory information of each case, which is accomplished, optimally, using systematic and standardized operating procedures, and in the framework of multidisciplinary teams with the support of a flexible and user-friendly database system that facilitates the sharing of information of all the teams in the network. We describe a biobanking system that is a platform for discovery research at the Center for Neurodegenerative Disease Research at the University of Pennsylvania.

URL:


## rPOP: Robust PET-only processing of community acquired heterogeneous amyloid-PET data.

The reference standard for amyloid-PET quantification requires structural MRI (sMRI) for preprocessing in both multi-site research studies and clinical trials. Here we describe rPOP (robust PET-Only Processing), a MATLAB-based MRI-free pipeline implementing non-linear warping and differential smoothing of amyloid-PET scans performed with any of the FDA-approved radiotracers (18F-florbetapir/FBP, 18F-florbetaben/FBB or 18F-flutemetamol/FLUTE). Each image undergoes spatial normalization based on weighted PET templates and data-driven differential smoothing, then allowing users to perform their quantification of choice. Prior to normalization, users can choose whether to automatically reset the origin of the image to the center of mass or proceed with the pipeline with the image as it is. We validate rPOP with n = 740 (514 FBP, 182 FBB, 44 FLUTE) amyloid-PET scans from the Imaging Dementia-Evidence for Amyloid Scanning - Brain Health Registry sub-study (IDEAS-BHR) and n = 1,518 scans from the Alzheimer's Disease Neuroimaging Initiative (n = 1,249 FBP, n = 269 FBB), including heterogeneous acquisition and reconstruction protocols. After running rPOP, a standard quantification to extract Standardized Uptake Value ratios and the respective Centiloids conversion was performed. rPOP-based amyloid status (using an independent pathology-based threshold of >=24.4 Centiloid units) was compared with either local visual reads (IDEAS-BHR, n = 663 with complete valid data and reads available) or with amyloid status derived from an MRI-based PET processing pipeline (ADNI, thresholds of >20/>18 Centiloids for FBP/FBB). Finally, within the ADNI dataset, we tested the linear associations between rPOP- and MRI-based Centiloid values. rPOP achieved accurate warping for N = 2,233/2,258 (98.9%) in the first pass. Of the N = 25 warping failures, 24 were rescued with manual reorientation and origin reset prior to warping. We observed high concordance between rPOP-based amyloid status and both visual reads (IDEAS-BHR, Cohen's k = 0.72 [0.7-0.74], ~86% concordance) or MRI-pipeline based amyloid status (ADNI, k = 0.88 [0.87-0.89], ~94% concordance). rPOP- and MRI-pipeline based Centiloids were strongly linearly related (R2:0.95, p<0.001), with this association being significantly modulated by estimated PET resolution (beta= -0.016, p<0.001). rPOP provides reliable MRI-free amyloid-PET warping and quantification, leveraging widely available software and only requiring an attenuation-corrected amyloid-PET image as input. The rPOP pipeline enables the comparison and merging of heterogeneous datasets and is publicly available at https://github.com/leoiacca/rPOP.

URL: https://github.com/leoiacca/rPOP.


## BioNOT: a searchable database of biomedical negated sentences.

BACKGROUND: Negated biomedical events are often ignored by text-mining applications; however, such events carry scientific significance. We report on the development of BioNOT, a database of negated sentences that can be used to extract such negated events. DESCRIPTION: Currently BioNOT incorporates  32 million negated sentences, extracted from over 336 million biomedical sentences from three resources:  2 million full-text biomedical articles in Elsevier and the PubMed Central, as well as  20 million abstracts in PubMed. We evaluated BioNOT on three important genetic disorders: autism, Alzheimer's disease and Parkinson's disease, and found that BioNOT is able to capture negated events that may be ignored by experts. CONCLUSIONS: The BioNOT database can be a useful resource for biomedical researchers. BioNOT is freely available at http://bionot.askhermes.org/. In future work, we will develop semantic web related technologies to enrich BioNOT.

URL: http://bionot.askhermes.org/.


## A benchmark for hypothalamus segmentation on T1-weighted MR images.

The hypothalamus is a small brain structure that plays essential roles in sleep regulation, body temperature control, and metabolic homeostasis. Hypothalamic structural abnormalities have been reported in neuropsychiatric disorders, such as schizophrenia, amyotrophic lateral sclerosis, and Alzheimer's disease. Although mag- netic resonance (MR) imaging is the standard examination method for evaluating this region, hypothalamic morphological landmarks are unclear, leading to subjec- tivity and high variability during manual segmentation. Due to these limitations, it is common to find contradicting results in the literature regarding hypothalamic volumetry. To the best of our knowledge, only two automated methods are available in the literature for hypothalamus segmentation, the first of which is our previous method based on U-Net. However, both methods present performance losses when predicting images from different datasets than those used in training. Therefore, this project presents a benchmark consisting of a diverse T1-weighted MR image dataset comprising 1381 subjects from IXI, CC359, OASIS, and MiLI (the latter created specifically for this benchmark). All data were provided using automatically generated hypothalamic masks and a subset containing manually annotated masks. As a baseline, a method for fully automated segmentation of the hypothalamus on T1-weighted MR images with a greater generalization ability is presented. The pro- posed method is a teacher-student-based model with two blocks: segmentation and correction, where the second corrects the imperfections of the first block. After using three datasets for training (MiLI, IXI, and CC359), the prediction performance of the model was measured on two test sets: the first was composed of data from IXI, CC359, and MiLI, achieving a Dice coefficient of 0.83; the second was from OASIS, a dataset not used for training, achieving a Dice coefficient of 0.74. The dataset, the baseline model, and all necessary codes to reproduce the experiments are available at https://github.com/MICLab-Unicamp/HypAST and https://sites.google.com/ view/calgary-campinas-dataset/hypothalamus-benchmarking. In addition, a leaderboard will be maintained with predictions for the test set submitted by anyone working on the same task.

URL: https://github.com/MICLab-Unicamp/HypAST


## Analysis of tauopathy research funding between 2006 and 2016 reveals critical gaps in research priorities.

Neurodegenerative diseases encompass a range of diagnoses, such as Alzheimer's disease and Parkinson's disease. Despite decades of advancements in understanding the neurobiology of individual diseases, this class has few disease-modifying therapeutics and a paucity of biomarkers for diagnosis or progression. However, tau protein aggregation has emerged as a potential unifying factor across several neurodegenerative diseases, which has prompted a rapid growth in tau-related funding. In spite of this growth, research funding in this area is not in line with the immense magnitude of disease burden, and drug discovery and clinical research remain underfunded. Coordinated, collaborative efforts are key to making an impact, which can and should be led by the major funding bodies within the tau space. Here we describe the development and analysis of a tau-focused neurodegeneration funding database, which captures data from 2040 grants from 2006 to 2016. This database was developed as a public resource to allow funders, researchers, and policy makers to better understand tau funding patterns and to identify key funders and potential collaborations. This database can be used in conjunction with other neurodegenerative disease databases, such as the International Alzheimer's Disease Research Portfolio to gain specific insight into tau-research funding. Over the study period, overall tau funding rose dramatically; however, changes in capital distribution also changed. Specifically, the field experienced a strong bias toward funding tau in the context of Alzheimer's disease, while at the same time generally decreasing the overall proportion of funding for basic research, treatment development, and evaluation. As funding organizations look forward, this resource can both inform future funding strategies and priority areas and identify potential collaborative efforts with complementary funding organizations.

URL:


## Deep learning-based EEG analysis to classify normal, mild cognitive impairment, and dementia: Algorithms and dataset.

For automatic EEG diagnosis, this paper presents a new EEG data set with well-organized clinical annotations called Chung-Ang University Hospital EEG (CAUEEG), which has event history, patient's age, and corresponding diagnosis labels. We also designed two reliable evaluation tasks for the low-cost, non-invasive diagnosis to detect brain disorders: i) CAUEEG-Dementia with normal, mci, and dementia diagnostic labels and ii) CAUEEG-Abnormal with normal and abnormal. Based on the CAUEEG dataset, this paper proposes a new fully end-to-end deep learning model, called the CAUEEG End-to-end Deep neural Network (CEEDNet). CEEDNet pursues to bring all the functional elements for the EEG analysis in a seamless learnable fashion while restraining non-essential human intervention. Extensive experiments showed that our CEEDNet significantly improves the accuracy compared with existing methods, such as machine learning methods and Ieracitano-CNN (Ieracitano et al., 2019), due to taking full advantage of end-to-end learning. The high ROC-AUC scores of 0.9 on CAUEEG-Dementia and 0.86 on CAUEEG-Abnormal recorded by our CEEDNet models demonstrate that our method can lead potential patients to early diagnosis through automatic screening.

URL:


## Normative data for subcortical regional volumes over the lifetime of the adult human brain.

Normative data for volumetric estimates of brain structures are necessary to adequately assess brain volume alterations in individuals with suspected neurological or psychiatric conditions. Although many studies have described age and sex effects in healthy individuals for brain morphometry assessed via magnetic resonance imaging, proper normative values allowing to quantify potential brain abnormalities are needed. We developed norms for volumetric estimates of subcortical brain regions based on cross-sectional magnetic resonance scans from 2790 healthy individuals aged 18 to 94years using 23 samples provided by 21 independent research groups. The segmentation was conducted using FreeSurfer, a widely used and freely available automated segmentation software. Models predicting subcortical regional volumes of each hemisphere were produced including age, sex, estimated total intracranial volume (eTIV), scanner manufacturer, magnetic field strength, and interactions as predictors. The mean explained variance by the models was 48%. For most regions, age, sex and eTIV predicted most of the explained variance while manufacturer, magnetic field strength and interactions predicted a limited amount. Estimates of the expected volumes of an individual based on its characteristics and the scanner characteristics can be obtained using derived formulas. For a new individual, significance test for volume abnormality, effect size and estimated percentage of the normative population with a smaller volume can be obtained. Normative values were validated in independent samples of healthy adults and in adults with Alzheimer's disease and schizophrenia.

URL:


## Functional annotation of genomic variants in studies of late-onset Alzheimer's disease.

Motivation: Annotation of genomic variants is an increasingly important and complex part of the analysis of sequence-based genomic analyses. Computational predictions of variant function are routinely incorporated into gene-based analyses of rare-variants, though to date most studies use limited information for assessing variant function that is often agnostic of the disease being studied. Results: In this work, we outline an annotation process motivated by the Alzheimer's Disease Sequencing Project, illustrate the impact of including tissue-specific transcript sets and sources of gene regulatory information and assess the potential impact of changing genomic builds on the annotation process. While these factors only impact a small proportion of total variant annotations (~5%), they influence the potential analysis of a large fraction of genes (~25%). Availability and implementation: Individual variant annotations are available via the NIAGADS GenomicsDB, at https://www.niagads.org/genomics/ tools-and-software/databases/genomics-database. Annotations are also available for bulk download at https://www.niagads.org/datasets. Annotation processing software is available at http://www.icompbio.net/resources/software-and-downloads/. Supplementary information: Supplementary data are available at Bioinformatics online.

URL: https://www.niagads.org/genomics/


## ChiTaRS 5.0: the comprehensive database of chimeric transcripts matched with druggable fusions and 3D chromatin maps.

Chimeric RNA transcripts are formed when exons from two genes fuse together, often due to chromosomal translocations, transcriptional errors or trans-splicing effect. While these chimeric RNAs produce functional proteins only in certain cases, they play a significant role in disease phenotyping and progression. ChiTaRS 5.0 (http://chitars.md.biu.ac.il/) is the latest and most comprehensive chimeric transcript repository, with 111 582 annotated entries from eight species, including 23 167 known human cancer breakpoints. The database includes unique information correlating chimeric breakpoints with 3D chromatin contact maps, generated from public datasets of chromosome conformation capture techniques (Hi-C). In this update, we have added curated information on druggable fusion targets matched with chimeric breakpoints, which are applicable to precision medicine in cancers. The introduction of a new section that lists chimeric RNAs in various cell-lines is another salient feature. Finally, using text-mining techniques, novel chimeras in Alzheimer's disease, schizophrenia, dyslexia and other diseases were collected in ChiTaRS. Thus, this improved version is an extensive catalogue of chimeras from multiple species. It extends our understanding of the evolution of chimeric transcripts in eukaryotes and contributes to the analysis of 3D genome conformational changes and the functional role of chimeras in the etiopathogenesis of cancers and other complex diseases.

URL: http://chitars.md.biu.ac.il/


## Normative morphometric data for cerebral cortical areas over the lifetime of the adult human brain.

Proper normative data of anatomical measurements of cortical regions, allowing to quantify brain abnormalities, are lacking. We developed norms for regional cortical surface areas, thicknesses, and volumes based on cross-sectional MRI scans from 2713 healthy individuals aged 18 to 94 years using 23 samples provided by 21 independent research groups. The segmentation was conducted using FreeSurfer, a widely used and freely available automated segmentation software. Models predicting regional cortical estimates of each hemisphere were produced using age, sex, estimated total intracranial volume (eTIV), scanner manufacturer, magnetic field strength, and interactions as predictors. The explained variance for the left/right cortex was 76%/76% for surface area, 43%/42% for thickness, and 80%/80% for volume. The mean explained variance for all regions was 41% for surface areas, 27% for thicknesses, and 46% for volumes. Age, sex and eTIV predicted most of the explained variance for surface areas and volumes while age was the main predictors for thicknesses. Scanner characteristics generally predicted a limited amount of variance, but this effect was stronger for thicknesses than surface areas and volumes. For new individuals, estimates of their expected surface area, thickness and volume based on their characteristics and the scanner characteristics can be obtained using the derived formulas, as well as Z score effect sizes denoting the extent of the deviation from the normative sample. Models predicting normative values were validated in independent samples of healthy adults, showing satisfactory validation R2. Deviations from the normative sample were measured in individuals with mild Alzheimer's disease and schizophrenia and expected patterns of deviations were observed.

URL:


## ROSE: A Retinal OCT-Angiography Vessel Segmentation Dataset and New Model.

Optical Coherence Tomography Angiography (OCTA) is a non-invasive imaging technique that has been increasingly used to image the retinal vasculature at capillary level resolution. However, automated segmentation of retinal vessels in OCTA has been under-studied due to various challenges such as low capillary visibility and high vessel complexity, despite its significance in understanding many vision-related diseases. In addition, there is no publicly available OCTA dataset with manually graded vessels for training and validation of segmentation algorithms. To address these issues, for the first time in the field of retinal image analysis we construct a dedicated Retinal OCTA SEgmentation dataset (ROSE), which consists of 229 OCTA images with vessel annotations at either centerline-level or pixel level. This dataset with the source code has been released for public access to assist researchers in the community in undertaking research in related topics. Secondly, we introduce a novel split-based coarse-to-fine vessel segmentation network for OCTA images (OCTA-Net), with the ability to detect thick and thin vessels separately. In the OCTA-Net, a split-based coarse segmentation module is first utilized to produce a preliminary confidence map of vessels, and a split-based refined segmentation module is then used to optimize the shape/contour of the retinal microvasculature. We perform a thorough evaluation of the state-of-the-art vessel segmentation models and our OCTA-Net on the constructed ROSE dataset. The experimental results demonstrate that our OCTA-Net yields better vessel segmentation performance in OCTA than both traditional and other deep learning methods. In addition, we provide a fractal dimension analysis on the segmented microvasculature, and the statistical analysis demonstrates significant differences between the healthy control and Alzheimer's Disease group. This consolidates that the analysis of retinal microvasculature may offer a new scheme to study various neurodegenerative diseases.

URL:


## Reference standard space hippocampus labels according to the European Alzheimer's Disease Consortium-Alzheimer's Disease Neuroimaging Initiative harmonized protocol: Utility in automated volumetry.

INTRODUCTION: A harmonized protocol (HarP) for manual hippocampal segmentation on magnetic resonance imaging (MRI) has recently been developed by an international European Alzheimer's Disease Consortium-Alzheimer's Disease Neuroimaging Initiative project. We aimed at providing consensual certified HarP hippocampal labels in Montreal Neurological Institute (MNI) standard space to serve as reference in automated image analyses. METHODS: Manual HarP tracings on the high-resolution MNI152 standard space template of four expert certified HarP tracers were combined to obtain consensual bilateral hippocampus labels. Utility and validity of these reference labels is demonstrated in a simple atlas-based morphometry approach for automated calculation of HarP-compliant hippocampal volumes within SPM software. RESULTS: Individual tracings showed very high agreement among the four expert tracers (pairwise Jaccard indices 0.82-0.87). Automatically calculated hippocampal volumes were highly correlated (rL/R = 0.89/0.91) with gold standard volumes in the HarP benchmark data set (N = 135 MRIs), with a mean volume difference of 9% (standard deviation 7%). CONCLUSION: The consensual HarP hippocampus labels in the MNI152 template can serve as a reference standard for automated image analyses involving MNI standard space normalization.

URL:


## Automated deep learning segmentation of high-resolution 7 Tesla postmortem MRI for quantitative analysis of structure-pathology correlations in neurodegenerative diseases.

Postmortem MRI allows brain anatomy to be examined at high resolution and to link pathology measures with morphometric measurements. However, automated segmentation methods for brain mapping in postmortem MRI are not well developed, primarily due to limited availability of labeled datasets, and heterogeneity in scanner hardware and acquisition protocols. In this work, we present a high-resolution dataset of 135 postmortem human brain tissue specimens imaged at 0.3 mm3 isotropic using a T2w sequence on a 7T whole-body MRI scanner. We developed a deep learning pipeline to segment the cortical mantle by benchmarking the performance of nine deep neural architectures, followed by post-hoc topological correction. We evaluate the reliability of this pipeline via overlap metrics with manual segmentation in 6 specimens, and intra-class correlation between cortical thickness measures extracted from the automatic segmentation and expert-generated reference measures in 36 specimens. We also segment four subcortical structures (caudate, putamen, globus pallidus, and thalamus), white matter hyperintensities, and the normal appearing white matter, providing a limited evaluation of accuracy. We show generalizing capabilities across whole-brain hemispheres in different specimens, and also on unseen images acquired at 0.28 mm3 and 0.16 mm3 isotropic T2*w fast low angle shot (FLASH) sequence at 7T. We report associations between localized cortical thickness and volumetric measurements across key regions, and semi-quantitative neuropathological ratings in a subset of 82 individuals with Alzheimer's disease (AD) continuum diagnoses. Our code, Jupyter notebooks, and the containerized executables are publicly available at the project webpage (https://pulkit-khandelwal.github.io/exvivo-brain-upenn/).

URL: https://pulkit-khandelwal.github.io/exvivo-brain-upenn/


## A systematic integrated analysis of brain expression profiles reveals YAP1 and other prioritized hub genes as important upstream regulators in Alzheimer's disease.

INTRODUCTION: Profiling the spatial-temporal expression pattern and characterizing the regulatory networks of brain tissues are vital for understanding the pathophysiology of Alzheimer's disease (AD). METHODS: We performed a systematic integrated analysis of expression profiles of AD-affected brain tissues (684 AD and 562 controls). A network-based convergent functional genomic approach was used to prioritize possible regulator genes during AD development, followed by functional characterization. RESULTS: We generated a complete list of differentially expressed genes and hub genes of the transcriptomic network in AD brain and constructed a Web server (www.alzdata.org) for public access. Seventeen hub genes active at the early stages, especially YAP1, were recognized as upstream regulators of the AD network. Cellular assays proved that early alteration of YAP1 could promote AD by influencing the whole transcriptional network. DISCUSSION: Early expression disturbance of hub genes is an important feature of AD development, and interfering with this process may reverse the disease progression.

URL:


## AAgMarker 1.0: a resource of serological autoantigen biomarkers for clinical diagnosis and prognosis of various human diseases.

Autoantibodies are produced to target an individual's own antigens (e.g. proteins). They can trigger autoimmune responses and inflammation, and thus, cause many types of diseases. Many high-throughput autoantibody profiling projects have been reported for unbiased identification of serological autoantigen-based biomarkers. However, a lack of centralized data portal for these published assays has been a major obstacle to further data mining and cross-evaluate the quality of these datasets generated from different diseases. Here, we introduce a user-friendly database, AAgMarker 1.0, which collects many published raw datasets obtained from serum profiling assays on the proteome microarrays, and provides a toolbox for mining these data. The current version of AAgMarker 1.0 contains 854 serum samples, involving 136 092 proteins. A total of 7803 (4470 non-redundant) candidate autoantigen biomarkers were identified and collected for 12 diseases, such as Alzheimer's disease, Bechet's disease and Parkinson's disease. Seven statistical parameters are introduced to quantitatively assess these biomarkers. Users can retrieve, analyse and compare the datasets through basic search, advanced search and browse. These biomarkers are also downloadable by disease terms. The AAgMarker 1.0 is now freely accessible at http://bioinfo.wilmer.jhu.edu/AAgMarker/. We believe this database will be a valuable resource for the community of both biomedical and clinical research.

URL: http://bioinfo.wilmer.jhu.edu/AAgMarker/.


## A comparative atlas of single-cell chromatin accessibility in the human brain.

Recent advances in single-cell transcriptomics have illuminated the diverse neuronal and glial cell types within the human brain. However, the regulatory programs governing cell identity and function remain unclear. Using a single-nucleus assay for transposase-accessible chromatin using sequencing (snATAC-seq), we explored open chromatin landscapes across 1.1 million cells in 42 brain regions from three adults. Integrating this data unveiled 107 distinct cell types and their specific utilization of 544,735 candidate cis-regulatory DNA elements (cCREs) in the human genome. Nearly a third of the cCREs demonstrated conservation and chromatin accessibility in the mouse brain cells. We reveal strong links between specific brain cell types and neuropsychiatric disorders including schizophrenia, bipolar disorder, Alzheimer's disease (AD), and major depression, and have developed deep learning models to predict the regulatory roles of noncoding risk variants in these disorders.

URL:


## The Brain Chart of Aging: Machine-learning analytics reveals links between brain aging, white matter disease, amyloid burden, and cognition in the iSTAGING consortium of 10,216 harmonized MR scans.

INTRODUCTION: Relationships between brain atrophy patterns of typical aging and Alzheimer's disease (AD), white matter disease, cognition, and AD neuropathology were investigated via machine learning in a large harmonized magnetic resonance imaging database (11 studies; 10,216 subjects). METHODS: Three brain signatures were calculated: Brain-age, AD-like neurodegeneration, and white matter hyperintensities (WMHs). Brain Charts measured and displayed the relationships of these signatures to cognition and molecular biomarkers of AD. RESULTS: WMHs were associated with advanced brain aging, AD-like atrophy, poorer cognition, and AD neuropathology in mild cognitive impairment (MCI)/AD and cognitively normal (CN) subjects. High WMH volume was associated with brain aging and cognitive decline occurring in an  10-year period in CN subjects. WMHs were associated with doubling the likelihood of amyloid beta (Abeta) positivity after age 65. Brain aging, AD-like atrophy, and WMHs were better predictors of cognition than chronological age in MCI/AD. DISCUSSION: A Brain Chart quantifying brain-aging trajectories was established, enabling the systematic evaluation of individuals' brain-aging patterns relative to this large consortium.

URL:


## DRAW+SneakPeek: analysis workflow and quality metric management for DNA-seq experiments.

SUMMARY: We report our new DRAW+SneakPeek software for DNA-seq analysis. DNA resequencing analysis workflow (DRAW) automates the workflow of processing raw sequence reads including quality control, read alignment and variant calling on high-performance computing facilities such as Amazon elastic compute cloud. SneakPeek provides an effective interface for reviewing dozens of quality metrics reported by DRAW, so users can assess the quality of data and diagnose problems in their sequencing procedures. Both DRAW and SneakPeek are freely available under the MIT license, and are available as Amazon machine images to be used directly on Amazon cloud with minimal installation. AVAILABILITY: DRAW+SneakPeek is released under the MIT license and is available for academic and nonprofit use for free. The information about source code, Amazon machine images and instructions on how to install and run DRAW+SneakPeek locally and on Amazon elastic compute cloud is available at the National Institute on Aging Genetics of Alzheimer's Disease Data Storage Site (http://www.niagads.org/) and Wang lab Web site (http://wanglab.pcbi.upenn.edu/).

URL: http://www.niagads.org/


## Objective features of subjective cognitive decline in a United States national database.

INTRODUCTION: Functional and cognitive features of subjective cognitive decline (SCD) were identified in a longitudinal database from the National Alzheimer's Coordinating Center. METHODS: Cognitively normal older adults with (SCD+) and without (SCD-) self-reported memory complaints (N = 3915) were compared on (1) baseline Functional Assessment Questionnaire ratings, (2) baseline scores and longitudinal rate of change estimates from nine neuropsychological tests, and (3) final clinical diagnoses. RESULTS: SCD+ had higher baseline ratings of functional impairment, reduced episodic memory practice effects and poorer performance on neuropsychological tests of psychomotor speed and language, and higher frequencies of mild cognitive impairment and dementia diagnoses at the end of follow-up compared with the SCD-group. DISCUSSION: Subtle clinical features of SCD identified in this large cohort are difficult to detect at the individual level. More sensitive tests are needed to identify those with SCD who are vulnerable to cognitive decline and dementia.

URL:


## Design and validation of the ADNI MR protocol.

Phase four of the Alzheimer's Disease Neuroimaging Initiative (ADNI4) magnetic resonance imaging (MRI) protocols aim to maintain longitudinal consistency across two decades of data acquisition, while adopting new technologies. Here we describe and justify the study's design and targeted biomarkers. The ADNI4 MRI protocol includes nine MRI sequences. Some sequences require the latest hardware and software system upgrades and are continuously rolled out as they become available at each site. The main sequence additions/changes in ADNI4 are: (1) compressed sensing (CS) T1-weighting, (2) pseudo-continuous arterial spin labeling (ASL) on all three vendors (GE, Siemens, Philips), (3) multiple-post-labeling-delay ASL, (4) 1 mm3 isotropic 3D fluid-attenuated inversion recovery, and (5) CS 3D T2-weighted. ADNI4 aims to help the neuroimaging community extract valuable imaging biomarkers and provide a database to test the impact of advanced imaging strategies on diagnostic accuracy and disease sensitivity among individuals lying on the cognitively normal to impaired spectrum. HIGHLIGHTS: A summary of MRI protocols for phase four of the Alzheimer's Disease Neuroimaging Initiative (ADNI 4). The design and justification for the ADNI 4 MRI protocols. Compressed sensing and multi-band advances have been applied to improve scan time. ADNI4 protocols aim to streamline safety screening and therapy monitoring. The ADNI4 database will be a valuable test bed for academic research.

URL:


## Bayesian model reveals latent atrophy factors with dissociable cognitive trajectories in Alzheimer's disease.

We used a data-driven Bayesian model to automatically identify distinct latent factors of overlapping atrophy patterns from voxelwise structural MRIs of late-onset Alzheimer's disease (AD) dementia patients. Our approach estimated the extent to which multiple distinct atrophy patterns were expressed within each participant rather than assuming that each participant expressed a single atrophy factor. The model revealed a temporal atrophy factor (medial temporal cortex, hippocampus, and amygdala), a subcortical atrophy factor (striatum, thalamus, and cerebellum), and a cortical atrophy factor (frontal, parietal, lateral temporal, and lateral occipital cortices). To explore the influence of each factor in early AD, atrophy factor compositions were inferred in beta-amyloid-positive (Abeta+) mild cognitively impaired (MCI) and cognitively normal (CN) participants. All three factors were associated with memory decline across the entire clinical spectrum, whereas the cortical factor was associated with executive function decline in Abeta+ MCI participants and AD dementia patients. Direct comparison between factors revealed that the temporal factor showed the strongest association with memory, whereas the cortical factor showed the strongest association with executive function. The subcortical factor was associated with the slowest decline for both memory and executive function compared with temporal and cortical factors. These results suggest that distinct patterns of atrophy influence decline across different cognitive domains. Quantification of this heterogeneity may enable the computation of individual-level predictions relevant for disease monitoring and customized therapies. Factor compositions of participants and code used in this article are publicly available for future research.

URL:


## Home monitoring of daily living activities and prediction of agitation risk in a cohort of people living with dementia.

BACKGROUND: People living with dementia (PLWD) have an increased susceptibility to developing adverse physical and psychological events. Internet of Things (IoT) technologies provides new ways to remotely monitor patients within the comfort of their homes, particularly important for the timely delivery of appropriate healthcare. Presented here is data collated as part of the on-going UK Dementia Research Institute's Care Research and Technology Centre cohort and Technology Integrated Health Management (TIHM) study. There are two main aims to this work: first, to investigate the effect of the COVID-19 quarantine on the performance of daily living activities of PLWD, on which there is currently little research; and second, to create a simple classification model capable of effectively predicting agitation risk in PLWD, allowing for the generation of alerts with actionable information by which to prevent such outcomes. METHOD: A within-subject, date-matched study was conducted on daily living activity data using the first COVID-19 quarantine as a natural experiment. Supervised machine learning approaches were then applied to combined physiological and environmental data to create two simple classification models: a single marker model trained using ambient temperature as a feature, and a multi-marker model using ambient temperature, body temperature, movement, and entropy as features. RESULT: There are 102 PLWD total included in the dataset, with all patients having an established diagnosis of dementia, but with ranging types and severity. The COVID-19 study was carried out on a sub-group of 21 patient households. In 2020, PLWD had a significant increase in daily household activity (p = 1.40e-08), one-way repeated measures ANOVA). Moreover, there was a significant interaction between the pandemic quarantine and patient gender on night-time bed-occupancy duration (p = 3.00e-02, two-way mixed-effect ANOVA). On evaluating the models using 10-fold cross validation, both the single and multi-marker model were shown to balance precision and recall well, having F1-scores of 0.80 and 0.66, respectively. CONCLUSION: Remote monitoring technologies provide a continuous and reliable way of monitoring patient day-to-day wellbeing. The application of statistical analyses and machine learning algorithms to combined physiological and environmental data has huge potential to positively impact the delivery of healthcare for PLWD.

URL:


## GENEVIC: GENetic data exploration and visualization via intelli- gent interactive console.

SUMMARY: The vast generation of genetic data poses a significant challenge in efficiently uncovering valuable knowledge. Introducing GENEVIC, an AI-driven chat framework that tackles this challenge by bridging the gap between genetic data generation and biomedical knowledge discovery. Leveraging generative AI, notably ChatGPT, it serves as a biologist's 'copilot'. It automates the analysis, retrieval, and visualization of customized domain-specific genetic information, and integrates functionalities to generate protein interaction networks, enrich gene sets, and search scientific literature from PubMed, Google Scholar, and arXiv, making it a comprehensive tool for biomedical research. In its pilot phase, GENEVIC is assessed using a curated database that ranks genetic variants associated with Alzheimer's disease, schizophrenia, and cognition, based on their effect weights from the Polygenic Score (PGS) Catalog, thus enabling researchers to prioritize genetic variants in complex diseases. GENEVIC's operation is user-friendly, accessible without any specialized training, secured by Azure OpenAI's HIPAA-compliant infrastructure, and evaluated for its efficacy through real-time query testing. As a prototype, GENEVIC is set to advance genetic research, enabling informed biomedical decisions. AVAILABILITY AND IMPLEMENTATION: GENEVIC is publicly accessible at https://genevic- anath2024.streamlit.app. The underlying code is open-source and available via GitHub at https://github.com/bsml320/GENEVIC.git (also at https://github.com/anath2110/GENEVIC.git). SUPPLEMENTARY INFORMATION: Available at Bioinformatics online and at https://github.com/bsml320/GENEVIC_Supplementary.git (also at https://github.com/anath2110/GENEVIC_Supplementary.git).

URL: https://genevic-


## Alzheimer's disease first symptoms are age dependent: Evidence from the NACC dataset.

INTRODUCTION: Determining the relationship between age and Alzheimer's disease (AD) presentation is important to improve understanding and provide better patient services. METHODS: We used AD patient data (N = 7815) from the National Alzheimer Coordinating Center database and multinomial logistic regression to investigate presentation age and first cognitive/behavioral symptoms. RESULTS: The odds of having a nonmemory first cognitive symptom (including impairment in judgment and problem solving, language, and visuospatial function) increased with younger age (P < .001, all tests). Compared with apathy/withdrawal, the odds of having depression and "other" behavioral symptoms increased with younger age (P < .02, both tests), whereas the odds of having psychosis and no behavioral symptom increased with older age (P < .001, both tests). DISCUSSION: There is considerable heterogeneity in the first cognitive/behavioral symptoms experienced by AD patients. Proportions of these symptoms change with age with patients experiencing increasing nonmemory cognitive symptoms and more behavioral symptoms at younger ages.

URL:


## Retention of American Indian and Alaska Native participants in the National Alzheimer's Coordinating Center Uniform Data Set.

INTRODUCTION: The number of American Indian and Alaska Native (AI/AN) elders is expected to double by 2060. Thus it is imperative to retain AI/AN participants in longitudinal research studies to identify novel risk factors and potential targets for intervention for Alzheimer's disease and related dementias in these communities. METHODS: The National Alzheimer's Coordinating Center houses uniformly collected longitudinal data from the network of National Institute on Aging (NIA)-funded Alzheimer's Disease Research Centers (ADRCs). We used logistic regression to quantify participant retention at 43 ADRCs, comparing self-identified AI/AN participants to non-Hispanic White (NHW) participants, adjusting for potential confounding factors including baseline diagnosis, age, sex, education, and smoking. RESULTS: The odds of AI/AN participant retention at the first follow-up visit were significantly lower than those for NHW participants (adjusted odds ratio [aOR]: 0.599; 95%: 0.46-0.78; p < 0.001). DISCUSSION: These results suggest the need for improved strategies to retain AI/AN participants, perhaps including improved researcher-community relationships and community engagement and education. HIGHLIGHTS: American Indian and Alaska Native (AI/AN) research participants were retained to the first follow-up appointment at lower rates than non-Hispanic White (NHW) participants. AI/AN participants are retained at lower rates than NHW participants for long-term follow-up. The majority of AI/AN participants were not retained to the second follow-up visit.

URL:


## SynthSR: A public AI tool to turn heterogeneous clinical brain scans into high-resolution T1-weighted images for 3D morphometry.

Every year, millions of brain magnetic resonance imaging (MRI) scans are acquired in hospitals across the world. These have the potential to revolutionize our understanding of many neurological diseases, but their morphometric analysis has not yet been possible due to their anisotropic resolution. We present an artificial intelligence technique, "SynthSR," that takes clinical brain MRI scans with any MR contrast (T1, T2, etc.), orientation (axial/coronal/sagittal), and resolution and turns them into high-resolution T1 scans that are usable by virtually all existing human neuroimaging tools. We present results on segmentation, registration, and atlasing of >10,000 scans of controls and patients with brain tumors, strokes, and Alzheimer's disease. SynthSR yields morphometric results that are very highly correlated with what one would have obtained with high-resolution T1 scans. SynthSR allows sample sizes that have the potential to overcome the power limitations of prospective research studies and shed new light on the healthy and diseased human brain.

URL:


## Brain clocks capture diversity and disparities in aging and dementia across geographically diverse populations.

Brain clocks, which quantify discrepancies between brain age and chronological age, hold promise for understanding brain health and disease. However, the impact of diversity (including geographical, socioeconomic, sociodemographic, sex and neurodegeneration) on the brain-age gap is unknown. We analyzed datasets from 5,306 participants across 15 countries (7 Latin American and Caribbean countries (LAC) and 8 non-LAC countries). Based on higher-order interactions, we developed a brain-age gap deep learning architecture for functional magnetic resonance imaging (2,953) and electroencephalography (2,353). The datasets comprised healthy controls and individuals with mild cognitive impairment, Alzheimer disease and behavioral variant frontotemporal dementia. LAC models evidenced older brain ages (functional magnetic resonance imaging: mean directional error = 5.60, root mean square error (r.m.s.e.) = 11.91; electroencephalography: mean directional error = 5.34, r.m.s.e. = 9.82) associated with frontoposterior networks compared with non-LAC models. Structural socioeconomic inequality, pollution and health disparities were influential predictors of increased brain-age gaps, especially in LAC (R2 = 0.37, F2 = 0.59, r.m.s.e. = 6.9). An ascending brain-age gap from healthy controls to mild cognitive impairment to Alzheimer disease was found. In LAC, we observed larger brain-age gaps in females in control and Alzheimer disease groups compared with the respective males. The results were not explained by variations in signal quality, demographics or acquisition methods. These findings provide a quantitative framework capturing the diversity of accelerated brain aging.

URL:


## PharmKG: a dedicated knowledge graph benchmark for bomedical data mining.

Biomedical knowledge graphs (KGs), which can help with the understanding of complex biological systems and pathologies, have begun to play a critical role in medical practice and research. However, challenges remain in their embedding and use due to their complex nature and the specific demands of their construction. Existing studies often suffer from problems such as sparse and noisy datasets, insufficient modeling methods and non-uniform evaluation metrics. In this work, we established a comprehensive KG system for the biomedical field in an attempt to bridge the gap. Here, we introduced PharmKG, a multi-relational, attributed biomedical KG, composed of more than 500 000 individual interconnections between genes, drugs and diseases, with 29 relation types over a vocabulary of ~8000 disambiguated entities. Each entity in PharmKG is attached with heterogeneous, domain-specific information obtained from multi-omics data, i.e. gene expression, chemical structure and disease word embedding, while preserving the semantic and biomedical features. For baselines, we offered nine state-of-the-art KG embedding (KGE) approaches and a new biological, intuitive, graph neural network-based KGE method that uses a combination of both global network structure and heterogeneous domain features. Based on the proposed benchmark, we conducted extensive experiments to assess these KGE models using multiple evaluation metrics. Finally, we discussed our observations across various downstream biological tasks and provide insights and guidelines for how to use a KG in biomedicine. We hope that the unprecedented quality and diversity of PharmKG will lead to advances in biomedical KG construction, embedding and application.

URL:


## Poststroke Seizures and the Risk of Dementia Among Young Stroke Survivors.

BACKGROUND: The impact of new onset seizures in young stroke survivors on the subsequent development of dementia is poorly understood. This study aimed to assess the association between new onset of seizure and dementia in a population-based study of stroke patients. METHODS: The IBM Watson Health MarketScan  Commercial Claims and Encounters database, for the years 2005-2014 served as the data source for this study. Using the International Classification of Diseases, Ninth Revision (ICD-9), we identified patients aged 18-60 years with ischemic strokes, IS (433.x1, 434.x1, and 436) and hemorrhagic strokes, HS (430, 431, 432.0, 432.1, and 432.9) between January 1, 2006, and December 31, 2009, which constituted our baseline study cohort. At baseline, all included participants were free of claims for dementia, brain tumors, toxin exposure, traumatic brain injury, and neuro-infectious diseases, identified using ICD-9 codes. They had at least 1-year continuous enrollment before the index stroke diagnosis and 5 years after, with no seizure claims within 1 year after the index date. The exposure of interest was seizures: a time-dependent variable. The study outcome of interest was dementia (ICD-9: 290.0, 290.10-13, 290.20-21, 290.3, 290.40-43, 291.2, 292.82, 294.10-11, 294.20-21, 294.8, 331.0, 331.11, 331.19, and 331.82), which occurred during the follow-up period from January 1, 2010, to December 31, 2014. A Cox proportional hazards regression model was applied to calculate the hazard ratio (HR) and 95% confidence interval (CI) for the independent association of seizures with the occurrence of dementia. FINDINGS: At the end of the baseline period, we identified 23,680 stroke patients (IS: 20,642 and HS: 3,038). The cumulative incidence of seizure was 6.7%, 6.4%, and 8.3% for all strokes, IS, and HS, respectively. The cumulative incidence of dementia was 1.3%, 1.4%, and 0.9% for all strokes, IS, and HS, respectively. After multivariable adjustment, young patients with stroke who developed seizures had a greater risk of dementia compared with those without seizures (All strokes adjusted HR: 2.53, 95%CI 1.84-3.48; IS: 2.52, 1.79-3.53; HS: 2.80, 1.05-7.43). CONCLUSION: These findings suggest that the onset of seizures in young stroke survivors is associated with a 2.53 times increased risk of developing dementia. CLASSIFICATION OF EVIDENCE: This study provides Class II evidence that post stroke seizures increase the probability of dementia in young stroke survivors.

URL:


## Locus coeruleus pathology is associated with cerebral microangiopathy at autopsy.

INTRODUCTION: We investigated the link between locus coeruleus (LC) pathology and cerebral microangiopathy in two large neuropathology datasets. METHODS: We included data from the National Alzheimer's Coordinating Center (NACC) database (n = 2197) and Religious Orders Study and Rush Memory and Aging Project (ROSMAP; n = 1637). Generalized estimating equations and logistic regression were used to examine associations between LC hypopigmentation and presence of cerebral amyloid angiopathy (CAA) or arteriolosclerosis, correcting for age at death, sex, cortical Alzheimer's disease (AD) pathology, ante mortem cognitive status, and presence of vascular and genetic risk factors. RESULTS: LC hypopigmentation was associated with higher odds of overall CAA in the NACC dataset, leptomeningeal CAA in the ROSMAP dataset, and arteriolosclerosis in both datasets. DISCUSSION: LC pathology is associated with cerebral microangiopathy, independent of cortical AD pathology. LC degeneration could potentially contribute to the pathways relating vascular pathology to AD. Future studies of the LC-norepinephrine system on cerebrovascular health are warranted. HIGHLIGHTS: We associated locus coeruleus (LC) pathology and cerebral microangiopathy in two large autopsy datasets. LC hypopigmentation was consistently related to arteriolosclerosis in both datasets. LC hypopigmentation was related to cerebral amyloid angiopathy (CAA) presence in the National Alzheimer's Coordinating Center dataset. LC hypopigmentation was related to leptomeningeal CAA in the Religious Orders Study and Rush Memory and Aging Project dataset. LC degeneration may play a role in the pathways relating vascular pathology to Alzheimer's disease.

URL:


## Spatially and temporally probing distinctive glycerophospholipid alterations in Alzheimer's disease mouse brain via high-resolution ion mobility-enabled sn-position resolved lipidomics.

Dysregulated glycerophospholipid (GP) metabolism in the brain is associated with the progression of neurodegenerative diseases including Alzheimer's disease (AD). Routine liquid chromatography-mass spectrometry (LC-MS)-based large-scale lipidomic methods often fail to elucidate subtle yet important structural features such as sn-position, hindering the precise interrogation of GP molecules. Leveraging high-resolution demultiplexing (HRdm) ion mobility spectrometry (IMS), we develop a four-dimensional (4D) lipidomic strategy to resolve GP sn-position isomers. We further construct a comprehensive experimental 4D GP database of 498 GPs identified from the mouse brain and an in-depth extended 4D library of 2500 GPs predicted by machine learning, enabling automated profiling of GPs with detailed acyl chain sn-position assignment. Analyzing three mouse brain regions (hippocampus, cerebellum, and cortex), we successfully identify a total of 592 GPs including 130 pairs of sn-position isomers. Further temporal GPs analysis in the three functional brain regions illustrates their metabolic alterations in AD progression.

URL:


## Voxel-based classification of FDG PET in dementia using inter-scanner normalization.

Statistical mapping of FDG PET brain images has become a common tool in differential diagnosis of patients with dementia. We present a voxel-based classification system of neurodegenerative dementias based on partial least squares (PLS). Such a classifier relies on image databases of normal controls and dementia cases as training data. Variations in PET image characteristics can be expected between databases, for example due to differences in instrumentation, patient preparation, and image reconstruction. This study evaluates (i) the impact of databases from different scanners on classification accuracy and (ii) a method to improve inter-scanner classification. Brain FDG PET databases from three scanners (A, B, C) at two clinical sites were evaluated. Diagnostic categories included normal controls (NC, nA=26, nB=20, nC=24 for each scanner respectively), Alzheimer's disease (AD, nA=44, nB=11, nC=16), and frontotemporal dementia (FTD, nA=13, nB=13, nC=5). Spatially normalized images were classified as NC, AD, or FTD using partial least squares. Supervised learning was employed to determine classifier parameters, whereby available data is sub-divided into training and test sets. Four different database setups were evaluated: (i) "in-scanner": training and test data from the same scanner, (ii) "x-scanner": training and test data from different scanners, (iii) "train other": train on both x-scanners, and (iv) "train all": train on all scanners. In order to moderate the impact of inter-scanner variations on image evaluation, voxel-by-voxel scaling was applied based on "ratio images". Good classification accuracy of on average 94% was achieved for the in-scanner setups. Accuracy deteriorated for setups with mismatched scanners (79-91%). Ratio-image normalization improved all results with mismatched scanners (85-92%). In conclusion, automatic classification of individual FDG PET in differential diagnosis of dementia is feasible. Accuracy can vary with respect to scanner or acquisition characteristics of the training image data. The adopted approach of ratio-image normalization has been demonstrated to effectively moderate these effects.

URL:


## Reproducible evaluation of classification methods in Alzheimer's disease: Framework and application to MRI and PET data.

A large number of papers have introduced novel machine learning and feature extraction methods for automatic classification of Alzheimer's disease (AD). However, while the vast majority of these works use the public dataset ADNI for evaluation, they are difficult to reproduce because different key components of the validation are often not readily available. These components include selected participants and input data, image preprocessing and cross-validation procedures. The performance of the different approaches is also difficult to compare objectively. In particular, it is often difficult to assess which part of the method (e.g. preprocessing, feature extraction or classification algorithms) provides a real improvement, if any. In the present paper, we propose a framework for reproducible and objective classification experiments in AD using three publicly available datasets (ADNI, AIBL and OASIS). The framework comprises: i) automatic conversion of the three datasets into a standard format (BIDS); ii) a modular set of preprocessing pipelines, feature extraction and classification methods, together with an evaluation framework, that provide a baseline for benchmarking the different components. We demonstrate the use of the framework for a large-scale evaluation on 1960 participants using T1 MRI and FDG PET data. In this evaluation, we assess the influence of different modalities, preprocessing, feature types (regional or voxel-based features), classifiers, training set sizes and datasets. Performances were in line with the state-of-the-art. FDG PET outperformed T1 MRI for all classification tasks. No difference in performance was found for the use of different atlases, image smoothing, partial volume correction of FDG PET images, or feature type. Linear SVM and L2-logistic regression resulted in similar performance and both outperformed random forests. The classification performance increased along with the number of subjects used for training. Classifiers trained on ADNI generalized well to AIBL and OASIS. All the code of the framework and the experiments is publicly available: general-purpose tools have been integrated into the Clinica software (www.clinica.run) and the paper-specific code is available at: https://gitlab.icm-institute.org/aramislab/AD-ML.

URL: https://gitlab.icm-institute.org/aramislab/AD-ML.
